docs/spec.md

```diff
@@ -6198,8 +6206,8 @@ def baseline_type(x: Value | Placeholder | Type) -> Type:
   if type(x) == TensorType:
     return x
   if type(x) == QuantizedTensorType:
-    element_type = quantized_element_type(x)
-    baseline_element_type = QuantizedElementType(
+    element_type = quantized_tensor_element_type(x)
```

Comment:
I think that we should replace most of the occurrences of `tensor_element_type` and `quantized_tensor_element_type` with just `element_type` to make things more concise. The more specific functions have additional functionality that ensures that the argument is of a specific type, but that's typically checked elsewhere, e.g. by input constraints that say "tensor" or "tensor or quantized tensor".

There are a few uses of more specific functions in this PR which I think are warranted:
  * `tensor_element_type` in the definition of `member_name`.
  * `tensor_element_type` and `quantized_element_type` in the definition of `element_type`.

Reply:
done